issue_template.md

```diff
@@ -1,29 +1,41 @@
-### Summary
-
--   What does this package do? (explain in 50 words or less):
+Submitting Author: Name (@github_handle)  
+Repository:   
+Version submitted:   
```

Comment:
depending on the decision on https://github.com/ropensci/onboarding-meta/issues/70 do we have submitters give the url to the git tagged version?

Reply:
Sounds fine. For now, we still ask for a URL and this update just reformats where in the workflow we ask for it. But I agree that with the URL + Version, we can ensure that reviewers are not chasing a moving target.